app/src/main/kotlin/dev/patrickgold/florisboard/ime/editor/EditorInstance.kt

```diff
@@ -458,6 +460,16 @@ class EditorInstance(context: Context) : AbstractEditorInstance(context) {
         }
     }
 
+    fun tryPerformEnterCommitRaw(): Boolean {
+        return if (subtypeManager.activeSubtype.primaryLocale.language.startsWith("zh") && activeContent.composing.length > 0) {
```

Comment:
Note: will change to `supportsAutoSpace` after you merge the other PR. Maybe change the name of that? Or add another one and call it `supportsEnterCommitRaw`?